ts/features/fims/store/reducers/index.ts

```diff
@@ -17,7 +17,7 @@ export type FimsState = {
 
 const INITIAL_STATE: FimsState = {
   currentFlowState: "consents",
-  consentsData: pot.none
+  consentsData: pot.noneLoading
```

Comment:
I think it is better to leave the state as `pot.none` and change the `isLoading` selector as to also consider `isStrictNone` as a loading state for `consents` flow state